### ISSUE TITLE
0.138.x causes go-getter syntax to fail

### ISSUE THREAD

**Issue by kevin-lindsay-1** [roboll/helmfile#1664]
When using version `0.138.x`, when using go-getter syntax the following is produced:
```
Error: non-absolute URLs should be in form of repo_name/path_to_chart, got: git::https://...
```

When I switch back to `0.137.0`, the error does not occur.

Helm version: `3.5.1`

**Comment by mumoshu** [comment]
@kevin-lindsay-1 Thanks for reporting!

This seems like a regression introduced via https://github.com/roboll/helmfile/pull/1629/files#diff-7e4e87a1e777423a3d9a915e00fb78a9c4fc2f4c476863f60b67e89b9c635697R990.
I'll certainly fix this in next patch version of helmfile.

**Comment by mumoshu** [comment]
This should be fixed via #1667

**Comment by mumoshu** [comment]
FYI v0.138.3 has been released for #1667

**Comment by mumoshu** [comment]
Closing this as resolved. Please reopen if necessary
